cmd/extender.go

```diff
@@ -0,0 +1,416 @@
+//
+// Copyright (c) 2021 SSH Communications Security Inc.
+//
+// All rights reserved.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/SSHcom/privx-sdk-go/api/authorizer"
+	"github.com/SSHcom/privx-sdk-go/api/userstore"
+	"github.com/spf13/cobra"
+)
+
+type extenderOptions struct {
+	accessGroupID   string
+	extenderID      string
+	fileName        string
+	clientType      string
+	trustedClientID string
+}
+
+func (m extenderOptions) normalizeClientType() string {
+	return strings.ToUpper(m.clientType)
+}
+
+func init() {
+	rootCmd.AddCommand(extenderCmd())
+}
+
+//
+//
+func extenderCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "extender",
```

Comment:
Based on the usage it seems that the top-most command should be "clients" or "trusted-clients" as it covers extender, web-proxy and carrier now.